### ISSUE TITLE
Poetry installs and uses a new python version instead of the existing one

### ISSUE THREAD

**Issue by vemonet** [python-poetry/poetry#6034]
- [x] I am on the [latest](https://github.com/python-poetry/poetry/releases/latest) Poetry version.
- [x] I have searched the [issues](https://github.com/python-poetry/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] If an exception occurs when executing a command, I executed it again in debug mode (`-vvv` option).

- **OS version and name**: Fedora 36
- **Poetry version**: 1.1.13
- **Link of a [Gist](https://gist.github.com/) with the contents of your pyproject.toml file**: https://github.com/sunchang0124/dp_cgans/blob/main/pyproject.toml

## Issue

Hi, I just installed poetry on my laptop (Fedora 36) using the automated install script from the official documentation: https://python-poetry.org/docs/

It worked, but `poetry` was installed with some confusing defaults 

I have python already installed (like everyone), which is version `3.9`. But poetry automatically installed python `3.10`, which I never asked for, and it sets itself to run with python `3.10`

To fix it and have poetry use my default python installed I needed to run the following command: 

```bash
poetry env use $(which python)
```

Those steps were not obvious from [the installation docs](https://python-poetry.org/docs/). Maybe I should not have used the recommended script and I should have directly installed it with `pip install` so that it would not have mess up with python versions. 
But I was thinking that the recommended script would be better designed to set up poetry, and between all the available options (pip install or script) it was not clear which one I should use (why one and not the other?)

I don't know why poetry automatically comes with python 3.10, but as a user I find it really confusing. I expect `poetry` to help me handling my python dependencies, and first thing it does is to create more confusion! Imo, it would make more sense to have poetry using the default python installation (is there any complication that I am not aware of that prevents to do this maybe?)

In conclusion:
* It would be beneficial for new users to clarify the installation process, especially regarding which python version will be used by poetry
* Ideally the poetry install script should use the default python installed (if it's with a supported version of course), or ask the user if they want to use their existing python, or install a new python 3.10 just for poetry

Thanks a lot for trying to make the python packaging ecosystem more sane!
